base/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV ENABLE_INIT_DAEMON true
 ENV INIT_DAEMON_BASE_URI http://identifier/init-daemon
 ENV INIT_DAEMON_STEP spark_master_init
 
-ENV SPARK_VERSION=2.1.1
+ENV SPARK_VERSION=2.1.2
 ENV HADOOP_VERSION=2.7
 
```

Comment:
Please, consider changing the path for getting the Spark binaries. It throws [an exception](http://d3kbcqa49mib13.cloudfront.net/spark-2.1.2-bin-hadoop2.7.tgz) : 

It is due 'The key does not exist on that mirror'. You could replace the row with : 
```sh 
https://archive.apache.org/dist/spark/spark-${SPARK_VERSION}/spark-${SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}.tgz
```

Reply:
Oh, sorry! Fixed it!